### PR TITLE
Add limitation for dataflows with administration mode

### PR DESCRIPTION
**Pull request by Kunter-Bunt**
Add limitation that dataflows must not be deployed to environments in admistration mode with a solution.

Observed behavior: The solution will import successfully, but the Dataflow experience won't show the Dataflow. The solution still shows it. This is detrimental because you can't configure the connection after the import.
Workaround: Uninstall solution, lift environment from administration mode, install solution again.

### CODE REVIEW

**Comment by Court72** [comment]
@DougKlopfenstein 

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Luitwieler** [comment]
@msftyehong is this expected behavior or something that we need to investigate as a bug?

**Comment by msftyehong** [comment]
@Kunter-Bunt , @Luitwieler I'm not aware that 'Administrator mode' alone blocks import. As far as I know it's the combination of admin mode with disabled 'Background operations' that prevents the successful import of Dataflows. Did you observe different?

(btw, it's a good idea to document it - thank you for suggesting it)

**Comment by Kunter-Bunt** [comment]
> @Kunter-Bunt , @Luitwieler I'm not aware that 'Administrator mode' alone blocks import. As far as I know it's the combination of admin mode with disabled 'Background operations' that prevents the successful import of Dataflows. Did you observe different? (i.e. admin mode with enabled background operations works as far as I know).
> 
> (btw, it's a good idea to document it - thank you for suggesting it)

That might be the case! I just observed that lifting the administration mode fixes it. But of course lifting admin mode enables background operation as well. 

So the limitation would then be "Dataflows can't be deployed to environments in where background operations are disabled."

File: powerquery-docs/dataflows/dataflow-solution-awareness.md [hunk 1]
```diff
@@ -123,3 +123,4 @@ For security reasons, credentials of connections used by dataflows aren't persis
 * To enable dataflow import, the Dataflows plugin performs unmanaged customizations during the asynchronous import step. These modifications are essential for a successful import.
 * Dataflows do not support block of unmanaged customizations. See, [Block unmanaged customizations in Dataverse environments](/power-platform/alm/block-unmanaged-customizations#known-limitations)
 * Dataflows do not support preferred solution. See, [Set a preferred solution](/power-apps/maker/data-platform/preferred-solution#limitations)
+* Dataflows can't be deployed to environments in administration mode.
```
Comment (msftyehong) [inline powerquery-docs/dataflows/dataflow-solution-awareness.md:126]:
```suggestion
* For dataflows to import successfully into an environment, it cannot be in administration mode, or background operations need to be enabled.
```

**Comment by msftyehong** [comment]
@Luitwieler once change is accepted and merged into PR branch, PR can merge into main branch.

File: powerquery-docs/dataflows/dataflow-solution-awareness.md (hunk 1 above)
Comment (DougKlopfenstein) [reply powerquery-docs/dataflows/dataflow-solution-awareness.md:126]: @msftyehong - the suggested change is a little ambiguous. Maybe "For dataflows to import successfully into an environment, either the environment can't be in administration mode, or background operations need to be enabled."
Let me know if that "either/or" is technically correct. The other changes were for Microsoft style.

Comment (msftyehong) [reply powerquery-docs/dataflows/dataflow-solution-awareness.md:126]: I modified my suggestion from "For dataflows to import successfully into an environment, it cannot be in administration mode, or background operations need to be enabled." to "For dataflows to import successfully into an environment, either the environment can't be in administration mode, or background operations need to be enabled."

**Review by Luitwieler — APPROVED** [review]

**Comment by Luitwieler** [comment]
#sign-off

**Comment by Court72** [comment]
@DougKlopfenstein - Is this PR ready for your review and signoff?

#hold-off to prevent merging

**Comment by DougKlopfenstein** [comment]
#sign-off

**Review by DougKlopfenstein — APPROVED** [review]

**Pull request merged**